bibtex/bib/quantum.bib

```diff
@@ -1328,6 +1415,19 @@ @Article{KW09
   Url                      = {http://dx.doi.org/10.1103/PhysRevLett.103.070504}
 }
 
+@Article{L02,
+  Title                    = {Quantum Vernam Cipher},
+  Author                   = {D. W. Leung},
```

Comment:
Done (also changed the key to "Leu02"). 
Out of curiosity, if the published version uses only first name initials, do you still expand it (by finding the full name elsewhere) in the BibTex?